examples/ffva/src/app_conf.h

```diff
@@ -220,4 +220,17 @@
 #define appconfINTENT_MODEL_RUNNER_TASK_PRIORITY  (configMAX_PRIORITIES - 2)
 #define appconfLED_TASK_PRIORITY                  (configMAX_PRIORITIES / 2 - 1)
 
+/* Software PLL settings for mclk recovery configurations */
+/* see fractions.h and register_setup.h for other pll settings */
+#define appconfLRCLK_NOMINAL_HZ     appconfI2S_AUDIO_SAMPLE_RATE
+#define appconfBCLK_NOMINAL_HZ      (appconfLRCLK_NOMINAL_HZ * 64)
+#define PLL_RATIO                   (MIC_ARRAY_CONFIG_MCLK_FREQ / appconfLRCLK_NOMINAL_HZ)
+#define PLL_CONTROL_LOOP_COUNT_UA   80   // How many SoF periods per control loop iteration. Aim for ~100Hz
```

Comment:
This shouldn't be needed - no UA

Reply:
I forgot to address this comment, I will do it in a separate PR.